ui/lib/pki/addon/components/page/pki-tidy-form.ts

```diff
@@ -4,33 +4,44 @@
  */
 
 import Component from '@glimmer/component';
+import errorMessage from 'vault/utils/error-message';
 import { action } from '@ember/object';
 import { inject as service } from '@ember/service';
 import { task } from 'ember-concurrency';
 import { waitFor } from '@ember/test-waiters';
+import { tracked } from '@glimmer/tracking';
 
-export default class PkiTidyForm extends Component {
-  @service router;
+import PkiTidyModel from 'vault/models/pki/tidy';
+import RouterService from '@ember/routing/router-service';
+
+interface Args {
+  tidy: PkiTidyModel;
+}
+
+export default class PkiTidyForm extends Component<Args> {
+  @service declare readonly router: RouterService;
+
+  @tracked errorBanner = '';
+  @tracked invalidFormAlert = '';
 
   returnToConfiguration() {
     this.router.transitionTo('vault.cluster.secrets.backend.pki.configuration.index');
   }
 
   @action
-  updateSafetyBuffer({ goSafeTimeString }) {
+  updateSafetyBuffer({ goSafeTimeString }: { goSafeTimeString: string }) {
     this.args.tidy.safetyBuffer = goSafeTimeString;
   }
 
   @task
   @waitFor
-  *save(e) {
-    e.preventDefault();
+  *save(event: Event) {
+    event.preventDefault();
     try {
       yield this.args.tidy.save();
       this.returnToConfiguration();
     } catch (e) {
-      const message = e.errors ? e.errors.join('. ') : e.message;
-      this.errorBanner = message;
+      this.errorBanner = errorMessage();
```

Comment:
you'll still want to pass the error object to the util here 
```suggestion
      this.errorBanner = errorMessage(e);
```